src/Aspire.Dashboard/Components/ResourcesGridColumns/StateColumnDisplay.razor

```diff
@@ -0,0 +1,44 @@
+﻿@using Aspire.Dashboard.Model
+@using Aspire.Dashboard.Otlp.Model
+@using Aspire.Dashboard.Resources
+@inject IStringLocalizer<Columns> Loc
+
+<div class="resource-state-container">
+    @Resource.Status?.State
+    @if (Resource.Status is { State: "Exited" /* containers */ or "Finished" /* executables */, ExitCode: not null and not 0 })
```

Comment:
@drewnoakes Should status still be a string if we're testing for known values in the UI?

Reply:
At the very least, known statuses should be centralized as const values on a type.

Also, when testing the value, we probably should ignore case (just in case). For example, have some methods such as `IsExited` and `IsFinished` that do a case insensitive check.